### DISCUSSION TITLE
manifestationsofmoloch

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#11] [Announcements]
# manifestationsofmoloch

The inescapable traps of Molochian systems through real-world examples like click-bait, bargain hunting, tax havens, and late-stage capitalism.

https://nonzerosum.games/manifestationsofmoloch.html

**Comment by leafstrat** [comment]
Your blog design is simply incredible!
So much soul.

**Reply by jamesstephenbrown** [reply]
Wow, thanks what a nice comment :) I'm trying to do something a bit different but with a unified look, so I'm glad you like it.
